src/test/java/GildedRoseTest.java

```diff
@@ -3,10 +3,155 @@
 
 class GildedRoseTest {
     @Test
-    void foo() {
-        Item[] items = new Item[] { new Item("foo", 0, 0) };
+    void sulfurasValuesAreConstant() {
+        Item[] items = new Item[] { new Item("Sulfuras, Hand of Ragnaros", 0, 80) };
         GildedRose app = new GildedRose(items);
         app.updateQuality();
-        assertEquals("foo", app.items[0].name);
+        assertEquals(0, app.items[0].sellIn);
+        assertEquals(80, app.items[0].quality);
+    }
+
+    @Test
+    void backstageItemsMaxQualityIs50() {
+        Item[] items = new Item[] {
+            new Item("Backstage passes to a TAFKAL80ETC concert", 5, 50)
+        };
+        GildedRose app = new GildedRose(items);
+        app.updateQuality();
+        assertEquals(50, app.items[0].quality);
+    }
+
+    @Test
+    void backstagePassesQualityGoesToZeroWhenSellInLessThanZero() {
+        Item[] items = new Item[] {
+                new Item("Backstage passes to a TAFKAL80ETC concert", 0, 20)
+        };
+        GildedRose app = new GildedRose(items);
+        app.updateQuality();
+        assertEquals(0, app.items[0].quality);
+    }
+
+    @Test
+    void backstagePassesQualityIncreasesByOneWhenSellInGreaterThanTen() {
+        Item[] items = new Item[] {
+                new Item("Backstage passes to a TAFKAL80ETC concert", 11, 20)
+        };
+        GildedRose app = new GildedRose(items);
+        app.updateQuality();
+        assertEquals(21, app.items[0].quality);
+    }
+
+    @Test
+    void backstagePassesQualityIncreasesByTwoWhenSellInBetweenFiveAndTen() {
+        Item[] items = new Item[] {
+                new Item("Backstage passes to a TAFKAL80ETC concert", 6, 20),
+                new Item("Backstage passes to a TAFKAL80ETC concert", 10, 20),
+        };
+        GildedRose app = new GildedRose(items);
+        app.updateQuality();
+        assertEquals(22, app.items[0].quality);
+        assertEquals(22, app.items[1].quality);
+    }
+
+    @Test
+    void backstagePassesQualityIncreasesByThreeWhenSellInBetweenFiveAndZero() {
+        Item[] items = new Item[] {
+                new Item("Backstage passes to a TAFKAL80ETC concert", 5, 20),
+                new Item("Backstage passes to a TAFKAL80ETC concert", 1, 20)
+        };
+        GildedRose app = new GildedRose(items);
+        app.updateQuality();
+        assertEquals(23, app.items[0].quality);
+        assertEquals(23, app.items[0].quality);
```

Comment:
I think y'all meant to make an assertion for the second item here 🙂

Reply:
Yes thank you